tooling/build/scripts/preBuild.sh

```diff
@@ -15,7 +15,7 @@ curl https://raw.githubusercontent.com/opengovsg/isomer-next/main/tooling/templa
 #######################
 
 # Temporary until we start doing proper releases of the Isomer components
-curl -L https://raw.githubusercontent.com/opengovsg/isomer-next/main/docs/isomerpages-isomer-components-0.0.13.tgz -o isomerpages-isomer-components-0.0.13.tgz
+curl -L hhttps://github.com/isomerpages/isomer-components-package/raw/main/opengovsg-isomer-components-0.0.13.tgz -o isomerpages-isomer-components-0.0.13.tgz
```

Comment:
better to use [raw.githubusercontent](https://stackoverflow.com/questions/39065921/what-do-raw-githubusercontent-com-urls-represent) bah